src/anyvar/restapi/main.py

```diff
@@ -0,0 +1,235 @@
+"""Provide core route definitions for REST service."""
+import ga4gh.vrs
+from fastapi import Body, FastAPI, HTTPException, Query, Request
+from pydantic import StrictStr
+
+import anyvar
+from anyvar.anyvar import AnyVar, create_storage, create_translator
+from anyvar.restapi.schema import (AnyVarStatsResponse, EndpointTag,
+                                   GetSequenceLocationResponse,
+                                   GetVariationResponse, InfoResponse,
+                                   RegisterVariationRequest,
+                                   RegisterVariationResponse, SearchResponse,
+                                   VariationStatisticType)
+from anyvar.translate.translate import TranslationException
+
+app = FastAPI(
+    version=anyvar.__version__,
+    docs_url="/",
+    openapi_url="/openapi.json",
+    swagger_ui_parameters={"tryItOutEnabled": True},
+    description="Register and retrieve VRS value objects."
+)
+
+@app.on_event("startup")
+async def startup():
+    """Get FUSOR reference"""
+    storage = create_storage()
+    translator = create_translator()
+    anyvar_instance = AnyVar(object_store=storage, translator=translator)
+    app.state.anyvar = anyvar_instance
+
+
+@app.get(
+    "/info",
+    response_model=InfoResponse,
+    summary="Check system status and configurations",
+    description="System status check and configurations",
+    tags=[EndpointTag.GENERAL]
+)
+def get_info():
+    """Get system status check and configuration"""
+    return {
+        "anyvar": {
+            "version": anyvar.__version__,
+            },
+        "ga4gh_vrs": {
+            "version": ga4gh.vrs.__version__
+        },
+    }
+
+
+@app.get(
+    "/locations/{location_id}",
+    response_model=GetSequenceLocationResponse,
+    summary="Retrieve sequence location",
+    description="Retrieve registered sequence location by ID",
+    tags=[EndpointTag.LOCATIONS]
+)
+def get_location_by_id(
+    request: Request,
+    location_id: StrictStr = Query(..., description="Location VRS ID")
```

Comment:
With the current setup, I get this error `AssertionError: Cannot use `Query` for path param 'location_id'`. Had to fix the other endpoints that used path parameters as well. 
```suggestion
    location_id: StrictStr
```

Reply:
Okay, I'm able to run `uvicorn anyvar.restapi.main:app --reload` after fixing above. It might be nice to include this command in the readme.

Reply:
@jsstevenson I still get `AssertionError: Cannot use `Query` for path param 'location_id'`. Do you not get this?

Reply:
(I'll add a temp readme)

This definitely should be a Path not a Query param, but I don't get any of these assertion errors, and the tests don't trigger them either. What version of FastAPI and Uvicorn do you have in the env?

Reply:
@jsstevenson fastapi is 0.95.0 and uvicorn is 0.21.1